docs/_data/post-summary.json

```
@@ -65,6 +65,16 @@
       "applies": ".s-post-summary--content",
       "description": "An optional content excerpt truncated at 2 lines."
     },
+    {
+      "class": ".s-post-summary--content-excerpt__sm",
+      "applies": ".s-post-summary--content-excerpt",
+      "description": "An optional content excerpt truncated at 1 line."
+    },
+    {
+      "class": ".s-post-summary--content-excerpt__lg",
+      "applies": ".s-post-summary--content-excerpt",
+      "description": "An optional content excerpt truncated at 3 lines."
+    },
```

I think `lg` is supposed to be 4 lines and we gotta document `s-post-summary--content-excerpt__md`, right?

```
        &.s-post-summary--content-excerpt__lg {
            .v-truncate4;
        }
```

```suggestion
    {
      "class": ".s-post-summary--content-excerpt__sm",
      "applies": ".s-post-summary--content-excerpt",
      "description": "An optional content excerpt truncated at 1 line."
    },
    {
      "class": ".s-post-summary--content-excerpt__md",
      "applies": ".s-post-summary--content-excerpt",
      "description": "An optional content excerpt truncated at 3 line."
    },
    {
      "class": ".s-post-summary--content-excerpt__lg",
      "applies": ".s-post-summary--content-excerpt",
      "description": "An optional content excerpt truncated at 4 lines."
    },
```